conf/default.php

```diff
@@ -38,7 +38,8 @@
 $conf['remote_brochure_path_mask'] = '/fykos%1$d/batch%2$d/out/serie%2$d.pdf';
 $conf['remote_serial_path_mask_cs'] = '/fykos%1$d/batch%2$d/out/serial%2$d.pdf';
 $conf['remote_serial_path_mask_en'] = '/fykos%1$d/batch%2$d/out/serial%2$d-en.pdf';
-$conf['remote_task_solution_path_mask'] = '/fykos%1$d/batch%2$d/out/reseni%2$d-%5$s.pdf';
+$conf['remote_task_solution_path_mask_cs'] = '/fykos%1$d/batch%2$d/out/solution%2$d-%5$s.pdf';
+$conf['remote_task_solution_path_mask_en'] = '/fykos%1$d/batch%2$d/out/solution%2$d-%5$s.pdf';
```

Comment:
Nemůže tohle rozbít staré ročníky? Nebo není potřeba přegenerovat celou Astrid? Starší ročníky mají pořád jen "reseni"

Reply:
Áno, môže ale len v prípade, že budeme chcieť sťahovať staré riešenia na web. 

Za mňa je to OK. :+1: